plugins/pi-memory/commands/promote-scan.md

```
@@ -0,0 +1,41 @@
+---
+description: Sweep the soul for promotion candidates — high-signal, durable, not-yet-promoted docs worth lifting to the knowledge system — and queue them for the gate
+---
+
+Scan the soul for documents worth promoting to a shared knowledge system, and present a ranked candidate list. This is the on-demand, in-session form of the pipeline's batch sweep: it automates the "mark a candidate" step across many docs at once, so you do not have to remember what is ready. It **never** promotes on its own — each candidate goes through `/promote`, which owns the human gate. (The fully autonomous batch sweep is a separate scheduled agent, not this command.)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)